packages/web-components/src/components/rux-classification-marking/test/index.html

```diff
@@ -17,8 +17,72 @@
         <script nomodule src="/build/astro-web-components.js"></script>
 
         <link rel="stylesheet" href="/build/astro-web-components.css" />
+
+        <script
```

Comment:
can we remove this? just so that it doesnt slow down our cypress tests trying to make an extra req